src/ai/commands/parsers/fast_transcription_command_parser.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.AI.Details.Common.CLI
+{
+    public class FastTranscriptionCommandParser : Parser
```

Comment:
you should look at the `recognize_command_parser.cs` to see how command parsing actually works...

Reply:
You'll also have to hook this up to `Speech_command_parser.cs` appropriately...